### PR TITLE
[ticket/15055] Display content on unexpted server error

### PR DESCRIPTION
**Pull request by marc1706**
PHPBB3-15055

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15055

### CODE REVIEW

File: tests/test_framework/phpbb_functional_test_case.php [hunk 1]
```diff
@@ -911,10 +911,15 @@ static public function assert_response_xml($status_code = 200)
...
 	* @param int $status_code	Expected status code
-	* @return null
+	* @return void
 	*/
 	static public function assert_response_status_code($status_code = 200)
 	{
+		if ($status_code != self::$client->getResponse()->getStatus() &&
+			preg_match('/^5[0-9]{2}/', self::$client->getResponse()->getStatus()))
+		{
+			self::fail('Encountered unexpected server error:\n' . self::$client->getResponse()->getContent());
```
Comment (Nicofuma) [inline tests/test_framework/phpbb_functional_test_case.php]: actually we might want to keep that

Comment (3D-I) [reply tests/test_framework/phpbb_functional_test_case.php]: Tracker ticket is wrong, that's the second case I noticed, takes always to "12345".

**Comment by 3D-I** [comment]
Tracker ticket is wrong, IMO.
Should be:
https://tracker.phpbb.com/browse/PHPBB3-15055

**Comment by marc1706** [comment]
Thx, currently trying to test why appveyor is having issues ;)

**Comment by marc1706** [comment]
@3D-I the pull request template has 12345 as default ticket ID. This obviously is supposed to be changed by the author (which I forgot in this case ;) ).

**Comment by 3D-I** [comment]
@marc1706 
Ah ok, that's why this one it was the second I found, always from you. ;))
I didn't know that, makes sense now.

**Pull request merged**